server/text_generation_server/utils/flash_attn.py

```diff
@@ -80,6 +85,25 @@ def attention(
     if window_size_left <= 0 and window_size_left != -1:
         raise ValueError("`window_size_left` must be > 0 or -1")
 
+    if IS_XPU_SYSTEM:
```

Comment:
This should raise a ValueError if window_size_left != -1 like for RocM.

Reply:
thanks, will update PR for it